Requests/Punchin.http

```diff
@@ -1,8 +1,8 @@
-@manager = eyJhbGciOiJIUzI1NiJ9.eyJlbXBsb3llZV9pZCI6MiwiZXhwIjoxNjczNDAwMjYzfQ.xhmu4zM5JJl3PrmpaH9Ml7eqZ16jzmdHA0jAkywk4eA
```

Comment:
You might want to `.gitignore` this file since it contains access tokens. It's not a huge deal in this case, but it's good to get into the practice of not committing any access tokens or secrets.